tasks/size_contracts.js

```diff
@@ -151,6 +168,8 @@ task(
   }
 
   console.log(table.toString());
+  if (config.outputFile)
```

Comment:
Maybe this would be better as a cli argument to the `size-contracts` task than as a config field?  Maybe both?

Reply:
My idea was to keep it consistent with `eth-gas-reporter`/`hardhat-gas-reporter`
If it was a cli argument then people using `runOnCompile` won't be able to take advantage of it I'm guessing
Both is a valid option IMO but I don't see any other arguments being passed via cli unless I'm missing something 🤔

Reply:
Leave as-is.  Might add a CLI argument later.